services/movie_session.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession:
+    if session_date:
+        date_object = datetime.strptime(session_date, "%Y-%m-%d").date()
+
+        return MovieSession.objects.filter(show_time__date=date_object)
+
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> MovieSession:
+    try:
+        movie_session = MovieSession.objects.get(id=session_id)
+
+        if show_time:
+            movie_session.show_time = show_time
+        if movie_id is not None:
+            movie_session.movie_id = movie_id
+        if cinema_hall_id is not None:
+            movie_session.cinema_hall_id = cinema_hall_id
+
+        movie_session.save()
+
+        return movie_session
+    except MovieSession.DoesNotExist:
+        raise ValueError(f"MovieSession with id {session_id} does not exist.")
+
+
+def delete_movie_session_by_id(movie_session_id: int) -> None:
+    try:
+        movie_session = MovieSession.objects.get(id=movie_session_id)
+        movie_session.delete()
+    except MovieSession.DoesNotExist:
+        raise ValueError(
+            f"MovieSession with id {movie_session_id} does not exist."
+        )
```

Comment:
1. You have def get_movie_session_by_id to get movie session by id
2. Just a tip for the future: I recommend to use .filter() instead of get because get will raise an error if movie session doesn't exist and you need always add additional try blocks and you won't be able to delete this record while using filter() will not raise an error + will delete nothing if object doesn't exist